src/platform/windows/main.cpp

```diff
@@ -327,11 +327,29 @@ int main(int argc, char** argv) {
             newEvent = false;
             if (inputManager->handleEvent(event)) continue;
             if (event.type == SDL_QUIT || (event.type == SDL_WINDOWEVENT && event.window.event == SDL_WINDOWEVENT_CLOSE)) running = false;
-            if (!inputManager->keyboardCaptured && event.type == SDL_KEYDOWN && event.key.repeat == 0) {
+
+            Key button;
```

Comment:
```c++

Key button = Key();
if (event.type == SDL_KEYDOWN && event.key.repeat == 0) {
    button = Key::keyboard(event.key.keysym.sym);
} else if (event.type == SDL_CONTROLLERBUTTONDOWN) {
    button = Key::controllerButton(event.cbutton);
} else if (event.type == SDL_CONTROLLERAXISMOTION) {
    button = Key::controllerMove(event.caxis);
}

if (!inputManager->keyboardCaptured && button.type != Key::Type::None) {
// (...)
```

Reply:
without checking button.controller.value this causes double firing